appinventor/appengine/src/com/google/appinventor/client/widgets/properties/SubsetJSONPropertyEditor.java

```diff
@@ -591,7 +591,7 @@ public void onValueChange(ValueChangeEvent<Boolean> valueChangeEvent) {
   }
 
   protected void updateValue() {
-    if (property.getValue() == "") {
+    if ("".equals(property.getValue())) {
```

Comment:
Consider making this `property.getValue() == null || property.getValue().isEmpty()`

Reply:
I was actually thinking about updating the com.google.appinventor.common.utils.StringUtils to include a method like Apache commons lang3 (IsEmpty) that checks for both null and length 0. Would that be better?

Reply:
@kkashi01 That sounds acceptable.

Reply:
@ewpatton Done. Added StringUtils.isEmpty and now using it